lua/rip-substitute/rg-operations.lua

```diff
@@ -1,12 +1,33 @@
 local M = {}
+local config = require("rip-substitute.config").config
```

Comment:
`config` should not be called at the top of the file, since it's then only set once. This prevents the user from running `.setup` again to change a config. Thus, `config` should always be set dynamically inside a function

Reply:
I was wondering about that. For some reason I feel kinda dirty requiring it on every execution of a function.
Is there any drawback to the way you're suggesting?

Reply:
no, since lua cache's is anyway.

Reply:
Ok great. That's an obvious choice then.